node/src/blockchain/blockchain_interface_utils.rs

```diff
@@ -326,36 +308,23 @@ pub fn send_payables_within_batch(
                     accounts,
                 ))
             }),
-    );
-}
-
-pub fn calculate_fallback_start_block_number(start_block_number: u64, max_block_count: u64) -> u64 {
-    if max_block_count == u64::MAX {
-        start_block_number + 1u64
-    } else {
-        start_block_number + max_block_count
-    }
+    )
 }
 
-pub fn convert_wei_to_gwei(wei: U256) -> u64 {
-    (wei / U256::from(GWEI_UNIT)).as_u64() + 1
-}
-
-pub fn create_blockchain_agent_web3(
-    gas_limit_const_part: u64,
+pub fn dynamically_create_blockchain_agent_web3(
```

Comment:
Hi Bert, Would you accept?

We used the word dynamically because we are returning a dyn.
and hope to stop anyone from refactoring this code by moving to the place its being called.